src/cpp/session/modules/SessionDataImportV2.R

```diff
@@ -13,6 +13,27 @@
 #
 #
 
+.rs.addFunction("path_relative_to_working_dir", function(target)
+{
+   if (is.null(target))
+   {
+      target
+   }
+   else
+   {
+      currentPath <- paste(getwd(), .Platform$file.sep, sep = "")
```

Comment:
@kevinushey if I run `.Platform$file.sep` in my Windows machine I still get `/`.

According to the docs (https://stat.ethz.ch/R-manual/R-devel/library/base/html/Platform.html): `"character string, giving the file separator used on your platform: "/" on both Unix-alikes and on Windows (but not on the former port to Classic Mac OS)." `

Should we leave this as is? Or do we have a preference to use `/` still?